### PR TITLE
Fix link - I made a mistake

### PR DESCRIPTION
**Pull request by Nestpebble**

### CODE REVIEW

File: docs/development/firmware/building.mdx [hunk 1]
```diff
@@ -62,7 +62,7 @@ The build system is modular. Adding a new board variant for an already supported
 
 1. Perform all of the steps [building with custom hardware](#build-with-custom-hardware) until your hardware runs fine.
 2. [Send a proposal](https://github.com/meshtastic/firmware/issues/new?assignees=&labels=enhancement%2Ctriage&template=New+Board.yml&title=%5BBoard%5D%3A+) to add a new board.
-3. If approved, go to (https://github.com/meshtastic/protobufs) and send a Pull Request for the `mesh.proto` file, adding your board to the `HardwareModel` Enum.
+3. If approved, go to <a href="https://github.com/meshtastic/protobufs">https://github.com/meshtastic/protobufs</a> and send a Pull Request for the `mesh.proto` file, adding your board to the `HardwareModel` Enum.
```
Comment (pdxlocations) [inline docs/development/firmware/building.mdx:65]: Any reason to not use markdown here?
[https://github.com/meshtastic/protobufs](https://github.com/meshtastic/protobufs)
Screenshot 2024-08-24 at 7 51 05 AM

File: docs/development/firmware/building.mdx [hunk 2]
```diff
@@ -72,7 +72,7 @@ The build system is modular. Adding a new board variant for an already supported
 
 Gitpod offers an alternative method for compiling the firmware without installing Visual Studio Code. It is a web browser-based online IDE that can be used with a GitHub account. Gitpod provides a free tier suitable for light use, such as compiling custom firmware.
 
-1. Go to (https://gitpod.io#https://github.com/meshtastic/firmware)[https://gitpod.io#https://github.com/meshtastic/firmware].
+1. Go to <a href="https://gitpod.io#https://github.com/meshtastic/firmware">https://gitpod.io#https://github.com/meshtastic/firmware</a>.
```
Comment (pdxlocations) [inline docs/development/firmware/building.mdx:75]: This one could use markdown too, there was just and extra space between the ) and the [

File: docs/development/firmware/building.mdx (hunk 1 above)
Comment (Nestpebble) [reply docs/development/firmware/building.mdx]: agreed - I'm getting carried away.

File: docs/development/firmware/building.mdx (hunk 2 above)
Comment (Nestpebble) [reply docs/development/firmware/building.mdx]: Square brackets go before round parentheses(?). I think it works now. 🤞

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**